### DISCUSSION TITLE
Which branch should I clone if I want to get the latest stable source code for .NET 7 or 8

### DISCUSSION THREAD

**Discussion by sujit1779** [umbraco/Umbraco-CMS#15336] [Features and ideas]
Hi, I want to clone the source code and see it for myself how Umbraco works. I see there are multiple branches not sure which one I should clone? I need the stable branch 12.3.3 version most probably because this is what I see in NUGET and this should run in .NET 7 (or .NET 8).
Thanks

**Comment by skttl** [comment]
You can find it by tags:
https://github.com/umbraco/Umbraco-CMS/releases/tag/release-7.15.11
https://github.com/umbraco/Umbraco-CMS/releases/tag/release-8.18.9

**Reply by c9mb** [reply]
I think the OP was asking about requiring .Net 7 or 8 - not that he wanted to use Umbraco 7 or 8

According to docs - [https://docs.umbraco.com/umbraco-cms/fundamentals/setup/requirements]() - version 12 requires .NET 7.0+
